server/email/email-controller.js

```diff
@@ -0,0 +1,55 @@
+var mandrill = require('./email-service');
+var _ = require('underscore');
+
+module.exports = function(app, io){
+  var models = app.get('models');
+  var Gigs = models.Gig;
+  var UserGigs = models.UserGigs;
+  var Users = models.User;
+  var Locations = models.Location;
+
+  var emailController = {
+
+    sendEmails: function (req, res) {
+      var gigId = req.body.gigId;
+
+      Gigs.getGigInfo(gigId).then(function (gigInfo) {
```

Comment:
this isn't how to promise chain. see below:

        var location;
        var gig;

        Gigs.getGigInfo(gigId).then(function (gigInfo) {
          gig = gigInfo;
          return Locations.getLocationInfo(gigInfo.LocationId);
        }).then(function(locationInfo){
          location = locationInfo;
          return UserGigs.getUserGigs(gigId)
        }).then(function(userIds){
          return models.sequelize.Promise.map(userIds, function(userId){
            return Users.getEmployeeEmail(userId.UserId);
          });
        }).then(function(userInfos){
          _.each(userInfos, function(userInfo){
            mandrill.sendEmployeeConfirmationMessage(gigInfo.dataValues, userInfo.dataValues, locationInfo.dataValues);
          }
        }).catch(function(err){
          console.log(err);
        })